data/services/aws-rds.toml

```diff
@@ -1,3 +1,5 @@
+# Service metadata lives
+# https://github.com/18F/aws-broker/blob/59f949b/catalog-template.yml#L8
```

Comment:
Nice idea! Suggest using the branch URLs - such as https://github.com/18F/aws-broker/blob/master/catalog-template.yml#L8